.github/workflows/main.yml

```diff
@@ -0,0 +1,57 @@
+name: Calendarium Scraper
+
+on:
+  schedule:
+    # run every 7 days
+    - cron: "0 0 * * 0"
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout Code
+        uses: actions/checkout@v4
+
+      - name: Setup Firefox
+        uses: browser-actions/setup-firefox@v1
+
+      - name: Firefox Version
+        run: firefox --version
```

Comment:
Why is this needed?

Reply:
It's only there for debugging purposes to inform which version of firefox was installed in the process. However, it is naturally not needed for the script to work.